### ISSUE TITLE
[Redesign] About WCAG-EM and conformance reporting content is @@@

### ISSUE THREAD

**Issue by mhansma96** [w3c/wcag-em-report-tool#485]
This is not a bug, because in the code it actually shows the '@@@', but it seems like content should be added here

**Comment by hidde** [comment]
thanks for filing! It was _just_ updated: https://github.com/w3c/wcag-em-report-tool/issues/447#issuecomment-862412357
